cores/arduino/HardwareTimer.h

```diff
@@ -90,6 +90,25 @@ typedef enum {
   PERCENT_COMPARE_FORMAT, // used for Dutycycle
 } TimerCompareFormat_t;
 
+typedef enum {
+  FILTER_NONE = 1,  // No filter
```

Comment:
Hi @dberlin 
Thanks for this pull request. It seems functionally good to me, but I wonder whether we can simplify it:
Can we get ride of function getICFBitsFromFilterMode() by setting `FILTER_NONE = 0`  ?
And thus enum `ChannelInputFilter_t `directly matches with ICF bit

Reply:
Done!